sdk/textanalytics/tests.yml

```diff
@@ -1,9 +1,16 @@
+parameters:
+- name: Location
+  displayName: Location
+  type: string
+  default: eastus
+
 trigger: none
 
 extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: textanalytics
+    Location: "${{ parameters.Location }}"
```

Comment:
FYI if you set this at the top level, the sovereign clouds (usgov/china) will not work because `eastus` is not a valid location for those clouds. I would nest `Location` under the `Public` object on line 16 instead.